documentation/modules/ROOT/pages/spring.adoc

```diff
@@ -32,6 +32,11 @@ Quarkus supports many of your favorite Spring Framework projects. In this chapte
 [INFO] ------------------------------------------------------------------------
 ----
 
+IMPORTANT: The extensions above require `quarkus-resteasy-jackson` or `quarkus-resteasy-reactive-jackson`.
+In order to be conservative with our dependencies, please change in `pom.xml` the extension `quarkus-reasteasy` 
+to `quarkus-resteasy-jackson`.
```

Comment:
Not sure I understand the remark here

Reply:
When we start our tutorial the project is generated with `quarkus-reasteasy` . However, when adding the Spring extensions you will notice an error message in the Dev Mode saying to you that they work with either `quarkus-resteasy-jackson` or `quarkus-resteasy-reactive-jackson`. This is why added this line here: to change in `pom.xml` from `quarkus-reasteasy` to `quarkus-resteasy-jackson`.

Reply:
Indeed, I just see it while testing ... Well that's an "interesting" feature that we had not before ... Isn't this a regression we should report to Quarkus ?